bibedit/record_book.xml

```diff
@@ -20,7 +20,12 @@ along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
 <!-- BibEdit-Template-Description: book -->
 <record>
   <datafield tag="020" ind1=" " ind2=" ">
-    <subfield code="a">VOLATILE:ISBN</subfield>
+    <subfield code="a">VOLATILE:ISBN (without dashes!)</subfield>
```

Comment:
shall we always recommend to put the 13digits one (978...)? Anyway we should write a BibCheck script that normalizes all isbn...